views/dashboard/blog-posts.ejs

```diff
@@ -17,7 +17,7 @@
 	</tr>
 	<% blogs.forEach(blog => { %>
 	<tr>
-		<td><%= `${blog.date.getDate()} ${months[blog.date.getMonth()]} ${blog.date.getFullYear()}` %></td>
+		<td><a href="/blog/clicked"><%= `${blog.date.getDate()} ${months[blog.date.getMonth()]} ${blog.date.getFullYear()}` %></a></td>
```

Comment:
I like the idea of linking to a page that displays a single blog from here, but but, because we're in the dashboard, such a link will be to edit the blog.